### ISSUE TITLE
scheme.show() crashes

### ISSUE THREAD

**Issue by keithbriggs** [nschloe/quadpy#387]
```

>>> print(quadpy.__version__)
0.16.2
>>> scheme.show()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/quadpy/s3/_helpers.py", line 29, in show
    balls=[((0.0, 0.0, 0.0), 1.0)],
  File "/usr/local/lib/python3.6/dist-packages/quadpy/helpers/plot.py", line 171, in show_vtk
    import vtk
ModuleNotFoundError: No module named 'vtk'
```

**Comment by nschloe** [comment]
Well, you need vtk installed.

**Comment by keithbriggs** [comment]
If vtk (never heard of it before) is a dependency, why doesn't pip install it?

**Comment by nschloe** [comment]
vtk is not installable from pypi. It's a large graphics library that comes with a Python front-end. You'll have to install it separately.

**Comment by keithbriggs** [comment]
That does not agree with what is said at https://pypi.org/project/vtk/: "pip install vtk".

________________________________

**Comment by nschloe** [comment]
Good call. Fixed by #389.
